### ISSUE TITLE
Add linter to detect input asking for password

### ISSUE THREAD

**Issue by mschwager** [dlint-py/dlint#16]
Bad:

```python
input("Password: ")
```

Good:

```python
getpass.getpass("Password: ")
```

See [getpass](https://docs.python.org/3/library/getpass.html).

We can probably just look for `input` calls that contain the string literal `'password'`.

**Comment by mschwager** [comment]
Note this may be somewhat moot since we're already searching for all `input` usage in `DUO108`. However, `six.moves.input` and `input` in Python > 3.0 are whitelisted, so probably still worth it.
